avocado/core/job.py

```diff
@@ -157,12 +157,18 @@ def __init__(self, config=None, test_suites=None):
         self.replay_sourcejob = self.config.get('replay_sourcejob')
         self.exitcode = exit_codes.AVOCADO_ALL_OK
 
+        self._res_eve_dis = None
```

Comment:
Don't be "shy" and use longer and more descriptive names here.  Something like `_result_events_dispatcher` is fine.

Reply:
Ok, will force push the name change